frontend/components/site/SiteUsers.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { SITE } from '../../propTypes';
+
+
+const SiteUsers = ({ site }) => {
+  // sort users by lower-cased usernames
+  const users = site.users.slice().sort((a, b) => {
+    const aName = a.username.toLowerCase();
+    const bName = b.username.toLowerCase();
+    if (aName === bName) { return 0; }
+    if (aName > bName) { return 1; }
+    return -1;
+  });
+
+  return (
+    <div>
+      <h4 className="label">Federalist users associated with this site</h4>
+      <table className="usa-table-borderless">
+        <thead>
+          <tr>
+            <th>User</th>
+          </tr>
+        </thead>
+        <tbody>
+          {users.map(user =>
+            (<tr key={user.username}>
+              <td>
+                <a
+                  href={`https://github.com/${user.username}`}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  title={`Visit GitHub profile for ${user.username}`}
+                >
+                  {user.username}
+                </a>
+                {' '}
+                {user.username.toLowerCase() === site.owner.toLowerCase() ? '(owner)' : ''}
```

Comment:
The `owner` badge is interesting to me. It looks like it is displayed next to the person who's GH the repo is on. That means for sites owned by an organization there is no owner.

I was confused by this for a bit b/c I did not connect to GH before looking at the code. I'm a little concerned about it confusing users. Interested in @wslack's thoughts.

Reply:
I see your point. We could definitely remove it!

What do you think about either demarcating or even not displaying the current user (like `jseppi` in the screenshot) in the table, since they'd be implied?

Reply:
That is also interesting 🤔. I like demarcating them, but not sure about removing them.

This will get a lot more interesting when we add the "Remove user" button.

Reply:
ok, I removed the "owner" badge and added a "you"  badge